src/vellum/workflows/runner/runner.py

```diff
@@ -143,6 +145,11 @@ def _run_work_item(self, node: BaseNode[StateType], span_id: UUID) -> None:
                         node_definition=node.__class__,
                         inputs=node._inputs,
                     ),
+                    parent=WorkflowParentContext(
+                        span_id=span_id,
+                        workflow_definition=self.workflow.__class__,
+                        parent=self._parent_context
+                    )
```

Comment:
Let's either use an execution context wrapper or DRY this out

Reply:
we're working on an execution context wrapper in a separate branch